README.md

```diff
@@ -143,6 +143,8 @@ There isn’t any fancy backend to this yet, so you’ll just have to add Doctor
     >>   phone_number: "02 9876 5432"
     >> )
 
+We've added some default contacts, the five we've currently got, to the db/seeds.rb file. This means that when the user runs rake db:setup, the database will already had the points of data declared in the seeds.rb file.
```

Comment:
This isn't really the proper use for seed data. As I understand it, seed data should include things that don't change and are standard to the app, e.g. “Default data”. For example, if you had a project that included a list of nations as objects, you might include them as seed data because they are unchanging. This is so that the data remains separate from from app logic.

As I understand it, seed data shouldn't be stuff that will need to be updated or changed. You'd have to reload all the data whenever you want to make a single change. This is bad because it means that whenever we change the values for one of these doctors, we need to remember to change it in the seed data as well as the database. This would be very easy to forget.

The current proper way to add data is in the readme here https://github.com/JasonThomasData/DocLink#adding-doctors-to-the-database . It's quite manual, will take a minute to do, but is better and more flexible. If it takes too much time, maybe it's time to open an issue for automatically loading data out of the spreadsheet. Here's how you run commands like `rails console` and `rake` on the remote Heroku db https://devcenter.heroku.com/articles/rake .

But if you find this useful for now, keep it, but I think it's gonna become a pain in the bum ;-).